### PR TITLE
Rework travis stages

### PR DESCRIPTION
**Pull request by PiotrZierhoffer**

### CODE REVIEW

**Comment by PiotrZierhoffer** [comment]
I'll merge it when travis passes (it should, but let's have a rerun)

**Review by mithro — APPROVED** [review]
LGTM!

File: .travis.yml [hunk 1]
```diff
@@ -4,60 +4,81 @@ addons:
   homebrew:
     packages:
     update: false
+
 stages:
-  - "Miscellaneous"
+  - name: "No dependencies" # Doesn't depend on anything apart from upstream packages
+    if: type != cron
```
Comment (mithro) [inline .travis.yml:10]: How often does the cron run? It might make sense to run the build regularly anyway?

**Comment by PiotrZierhoffer** [comment]
travis fails because macos, we'll fix it in a sec

**Pull request merged**